input/docs/getting-started/index.md

```diff
@@ -227,28 +198,26 @@ Now we need to create a View for our ViewModel, the following is an example:
         <Grid.ColumnDefinitions>
             <ColumnDefinition Width="Auto" />
             <ColumnDefinition Width="*" />
-            <ColumnDefinition Width="Auto" />
         </Grid.ColumnDefinitions>
         <Grid.RowDefinitions>
             <RowDefinition Height="Auto" />
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
-        <TextBlock FontSize="16" FontWeight="Bold" VerticalAlignment="Center">Search For:</TextBlock>
-        <TextBox Grid.Column="1" Margin="6,0,0,0" 
+        <TextBlock FontSize="16" FontWeight="SemiBold" VerticalAlignment="Center" Text="Search for:"/>
+        <TextBox Grid.Column="1" Margin="6 0 0 0" 
                  Text="{Binding SearchTerm, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"/>
-        <TextBlock Grid.Column="2" Margin="6,0,0,0" FontSize="16" FontWeight="Bold" Text="..." 
-                   Visibility="{Binding SpinnerVisibility, Mode=OneWay}" />
         <ListBox Grid.ColumnSpan="3" Grid.Row="1" Margin="0,6,0,0" 
                  ScrollViewer.HorizontalScrollBarVisibility="Disabled"
-                 ItemsSource="{Binding SearchResults, Mode=OneWay}" 
-                 ItemTemplate="{DynamicResource PhotoDataTemplate}"  />
+                 ItemsSource="{Binding SearchResults, Mode=OneWay}"
```

Comment:
We should be showing off Reactive UI Binding here.

Reply:
Hmm. Not sure if we should jump-start with complex features here. 
XAML bindings is something wpf/uwp/xf developers are familiar with.

Reply:
Reactive binding is one of the fundamental features of the framework. It's up there with whenanyvalue so we should be. Eg cross platform wouldn't work without it.

Reply:
If we use ReactiveUI bindings, then we should use `WhenActivated` and explain why we need it (static DependencyProperties, ViewModels that can potentially outlive Views); there is a `DataTemplate` there - so we need a `ReactiveUserControl`, also need to explain how it works.

I agree we could attract new users who come from `Xamarin.Android`, `Xamarin.iOS` or `Windows Forms` by showing them human-readable bindings that work on platforms that don't support XAML, but I'm not sure this should be a part of a simple Hello-World tutorial. 

Maybe we can say "Hey folks coming from XF Native or WinForms, look, we have something you'll definitely like" and redirect them to a guide that explains `Bind`, `BindTo`, `WhenActivated`, `ReactiveUserControl` and why ReactiveUI bindings are better than traditional XAML `{Binding Path, Converter={StaticResource Blablabla}}`

Reply:
Hmm but if Wyam supports spoilers that can expand/collapse, we can show and compare two different ways of doing the same thing 🤔

Reply:
Yeah see if it does maybe. I think you are showing roots of only being a wpf programmer on this subject though and this framework is much more. And it's not "complex"

Reply:
Well it can be if you go into value converters and stuff but let's not do that in a startup guide. Yeah if we can do expand box that would he ideal.

Reply:
And reactive binding should he used by wpf uwp programmers also due to the type safety and better value converter mechanisms. Our official advice is reactive binding is recommended over xaml bindings. So if we have a startup guide ignoring that advice not great.

Reply:
BTW in WPF/UWP land to give you an idea also how this works.

If you don't have a ItemTemplate on a class derived off ItemsControl (ListBox for example), and you bind to the the ItemSource, it'll use Splat to find a IViewFor<ItemsSourceItemType> and use those views inside your ListBox and provide the ItemTemplate for you.

Reply:
Like I said lets not complicate things with that magic. But something to keep in mind if you're looking at that documentation at some other point.

Reply:
>To keep things simple, keep your ItemTemplate with the Xaml Bindings and say "hey look if you need to we support XAML bindings too..."

I'm afraid mixing two ways of doing the same thing can lead to misunderstanding...

BTW just realized ReactiveUI repo now has IntegrationTests directory, and there is an *absolutely ideal* cross-platform app demonstrating almost all features ReactiveUI has - including `IViewFor`, `Bind`, `BindTo`, `WhenActivated` and more, so maybe we should simply say something like:

 — This is a very basic hello-world application demonstrating only a few features ReactiveUI framework has. If you would like to continue learning, take a look at our <a href="https://github.com/reactiveui/ReactiveUI/tree/master/integrationtests">IntegrationTests app</a> — it uses not only `WhenAny`, `OAPH` and `ReactiveCommand` features, but also <a href="https://reactiveui.net/docs/handbook/data-binding/">type-safe bindings</a>, <a href="https://reactiveui.net/docs/handbook/when-activated/">activation</a>, <a href="https://reactiveui.net/docs/handbook/commands/canceling">canceling</a> and <a href="https://reactiveui.net/docs/handbook/views/view-location">view location</a>. Also, it supports .NET Standard, Android, iOS, Xamarin.Forms, Universal Windows, Windows Forms and Windows Presentation Foundation.

Reply:
I'll update the sample, at the moment without the Bind() it's just not showing our recommended best practices.

Reply:
Ok, let it be so. Anyway, I think you could do it better than me as I have no experience with code-behind bindings. Thanks! :)

Reply:
I'll do it as another PR into your branch so you can see the changes. Might be worth btw just for your own knowledge just to try them out and the different scenarios. Try the IViewFor<> recognition stuff with a ListBox (by not setting the ItemTemplate)

Reply:
Sorry I am late to the party.  I agree that one of the key features that sets ReactiveUI apart are the `Bind()` extensions to interact with the binding engine.  If this is the getting started page, we should show consumers how to get started with the key features that makes ReactiveUI the composable framework that we love.